docs/src/components/ColorPalette.js

```diff
@@ -0,0 +1,43 @@
+// @flow strict
+import type { Node } from 'react';
+import { Box, Text } from 'gestalt';
+// $FlowFixMe[untyped-import]
```

Comment:
nit: this can be `$FlowExpectedError`, since I don't think we ever expect to fix this